empress_wrapper_examples.py

```diff
@@ -0,0 +1,74 @@
+# TODO: To be converted to automated tests
+# https://github.com/ssantichaivekin/eMPRess/issues/32
+import empress
+from matplotlib.backends.backend_tkagg import FigureCanvasTkAgg
+import matplotlib.pyplot as plt
+import tkinter
+
+# Read Reconciliation Input
+recon_input = empress.read_tree("./example/heliconius.newick")
+
+# Display Costscape Window where we can click on x/y axis
+class TkinterCostPolygonPage:
+    """Put this class in your app"""
+    def __init__(self):
+        # or put these lines in your class
+        cost_poly = empress.compute_cost_polygon(recon_input)
+        fig = cost_poly.draw()
+        canvas = FigureCanvasTkAgg(fig)
+        canvas.draw()
+
+# Compute ReconGraph
+recongraph = empress.reconcile(recon_input, 1, 3, 2)
+
+# Save Histogram
+recongraph.draw_to_file("./histogram_example.pdf")
+
+# Display Histogram
+class TkinterHistogramPage:
+    """Put this class in your app"""
+    def __init__(self):
+        # or put these lines in your class
+        fig = recongraph.draw()
+        canvas = FigureCanvasTkAgg(fig)
+        canvas.draw()
+
+# Get a random median from ReconGraph
+recon = recongraph.find_median()
+
+# Display one Reconciliation
+class TkinterReconPage:
+    """Put this class in your app"""
+    def __init__(self):
+        # or put these lines in your class
+        fig = recon.draw()
+        canvas = FigureCanvasTkAgg(fig)
+        canvas.draw()
+
+# Save Reconciliation drawing
+recon.draw_to_file("./recon_example.pdf")
+
+# Display Histogram of different clusters (by creating multiple matplotlib axes)
+class TkinterMultiHistogramPage:
+    """Put this class in your app"""
+    def __init__(self):
+        fig, axs = plt.subplots(3, 3)
```

Comment:
I think it's reasonable to ask front end developers to have a good understanding of matplotlib `Figure` and `Axes` and create the figures/axes instances themselves as shown in this line.

Reply:
I agree.  Note though that the subplots arguments won't necessarily be 3, 3.

Reply:
I agree. `(3,3)` is only an example :)